src/platform/tls_schannel.c

```diff
@@ -1076,6 +1115,47 @@ CxPlatTlsAchWorker(
 
 #endif
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+QUIC_STATUS
+CxPlatTlsSetClientCertPolicy(
+    _In_ CXPLAT_SEC_CONFIG* SecConfig
+    )
+{
+    SECURITY_STATUS SecStatus = SEC_E_OK;
+    SecPkgCred_ClientCertPolicy ClientCertPolicy;
+    CXPLAT_FRE_ASSERT(!(SecConfig->Flags & QUIC_CREDENTIAL_FLAG_CLIENT));
```

Comment:
I'd prefer a DBG assert here.

Reply:
Wouldn't this make a good TEL assert? Since this is something that eventually is gotten from user code?

Reply:
Not really, because the code that calls this function only does so when the flag is set.